### PR TITLE
[17.0][MIG] migration of module: ipushp

### PR DESCRIPTION
**Pull request by nirmal-serpentcs**
migrated module in v17 and made some changes in page view of registration and team member card view . added validation in form by js

### CODE REVIEW

**Comment by nirmal-serpentcs** [comment]
iPushp module is tested by Pooja Patel and module is ready to merge in v17 .

**Comment by ketanserpentcs** [comment]
Testing has been completed by Pooja Patel

File: ipushp/data/relation_data.xml
Comment (ChiragPatel-SerpentCS) [inline ipushp/data/relation_data.xml:1]: @nirmal-serpentcs 
Add a "Spouse" data too.

File: ipushp/data/website_data.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<odoo>
+    <data noupdate="1">
+        <record id="menu_website_ipushp" model="website.menu">
+            <field name="name">iPushp</field>
+            <field name="url">/page/iPushp</field>
```
Comment (ChiragPatel-SerpentCS) [inline ipushp/data/website_data.xml]: @nirmal-serpentcs 
here URL should be SEO friendly.
like : '/ipushp'

**Review by ChiragPatel-SerpentCS — APPROVED** [review]

**Pull request merged**